exploration/beauty-contest.md

```diff
@@ -0,0 +1,378 @@
+# Syntax Variation Beauty Contest
+
+As discussed in the 2023-09-25 teleconference, we need to choose a syntax to proceed with.
+This page hosts various options for considering in the 2023-10-02 call.
+
+## Shorthand description of some aspects of the options
+
+Please dispute factual errors in this. The table uses multiple `-` when an option requires more than one of something. `+` is not weighted.
+
+| Option | Description                                                    | Doesn’t Nest {} | Doesn’t Need More Escapes | Doesn’t Require Quoted Pattern | Allows Unquoted Literal Operands | Counted {} works | Adds no sigils |
+| :----- | :------------------------------------------------------------- | :-------------- | :------------------------ | :----------------------------- | :------------------------------- | :--------------- | :------------- |
+| 0      | Current syntax, starts in code mode                            | -               | +                         | --                             | +                                | -                | +              |
+| 1      | Invert for text mode                                           | -               | +                         | +                              | -                                | +                | +              |
+| 1a     | Invert for text mode, distinguish statements from placeholders | -               | +                         | +                              | +                                | +                | -              |
+| 2      | Text first, but always code mode after code                    | -               | +                         | -                              | -                                | +                | +              |
+| 3      | Use sigils for code mode                                       | +               | --                        | +                              | +                                | +                | --             |
```

Comment:
The escapes for 3 should be the same as 3a: `#` in patterns.

```suggestion
| 3      | Use sigils for code mode                                       | +               | -                         | +                              | +                                | +                | --             |
```

Reply:
3 also introduces a special use for `[`/`]`. Presumably we could work around unquoted literals by excluding that character from unquoted, so I'll merge this